styles/globals.css

```diff
@@ -282,3 +282,12 @@ body:has([aria-hidden="false"][data-star-search-mobile]) {
 body:has([data-star-search-mobile]) .top-nav-container {
   @apply fixed;
 }
+
+/**
+  * The pinned-table-column CSS class is for our tables that have pinned columns.
+  * When embedded StarSearch is open, we want to ensure that the columns unpin so they
+  * don't leak throug the embedded StarSearch panel.
+  */
+body:not(:has([data-star-search-mobile][aria-hidden="false"])) .pinned-table-column {
```

Comment:
This works when not in mobile view as well. This attribute, `data-star-search-mobile` always exists, it's just either `"true"` or `"false"` and the `[aria-hidden="false"]` indicates that the panel is open so we shouldn't pin the table columns.